fhir-examples/src/main/resources/json/ibm/fhir-operation-erase/Patient-1.json

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
are we going back to including test resources in each project vs pulliing them from fhir-examples?
i've gone back and forth on this in my head and either is probably fine, but lately i've been using them right out of fhir-examples instead of copying them over.

Reply:
This is in fhir-examples. I'm using ones I know work the way I need them to for the tests.

Reply:
lol, i totally missed that this was in fhir-examples 😊